doc/source/index.md

```diff
@@ -2,200 +2,226 @@
-## What documentation resource is right for you?
+### Parallel hyperparameter tuning experiments
+Running multiple hyperparameter tuning experiments is a pattern apt for distributed computing because each experiment is independent of one another. Ray Tune handles the hard bit of distributing hyperparameter optimization and makes available key features such as checkpointing the best result, optimizing scheduling, and specifying search patterns.
 
+|<img src="./images/tuning_use_case.png" width="70%" loading="lazy">|
+|:--:|
+|Distributed tuning with distributed training per trial.|
 
-````{panels}
-:container: text-center
-:column: col-lg-6 px-2 py-2
-:card:
+ See [hyperparameter tuning examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#hyperparameter-tuning) for use cases.
 
----
-**Getting Started**
+### Reinforcement learning
+Ray RLlib offers support for production-level, distributed reinforcement learning workloads while maintaining unified and simple APIs for a large variety of industry applications.
 
-<img src="images/getting_started.svg" alt="getting_started" height="40px">
-^^^^^^^^^^^^^^^
+|<img src="./images/rllib_use_case.png" width="70%" loading="lazy">|
+|:--:|
+|Decentralized distributed proximal polixy optimiation (DD-PPO) architecture.|
 
-If you're new to Ray, check out the getting started guide.
-You will learn how to install Ray, how to compute an example with the Ray Core API, and how to use each of Ray's ML libraries.
-You will also understand where to go from there.
+ See [reinforcement learning examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#reinforcement-learning) for use cases.
 
-+++
+### Batch inference on CPUs and GPUs
+Performing inference on incoming batches of data can be parallelized by exporting the architecture and weights of a trained model to the shared object store. Using these model replicas, Ray scales predictions on batches across workers.
 
-{link-badge}`ray-overview/index.html,"Getting Started",cls=badge-light`
+|<img src="./images/batch_inference.png" width="70%" loading="lazy">|
+|:--:|
+|Using Ray AIR's `BatchPredictor` for batch inference.|
 
----
-**User Guides**
+ See [batch inference examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#batch-inference) for use cases.
 
-<img src="images/user_guide.svg" alt="user_guide" height="40px">
-^^^^^^^^^^^
+### Multi-model composition for model serving
 
-Our user guides provide you with in-depth information about how to use Ray's libraries and tooling.
-You will learn about the key concepts and features of Ray and how to use them in practice.
-+++
+[Ray Serve](https://docs.ray.io/en/latest/serve/index.html) supports complex [model deployment patterns](https://www.youtube.com/watch?v=mM4hJLelzSw) requiring the orchestration of multiple Ray actors, where different actors provide inference for different models. Serve handles both batch and online inference and can scale to thousands of models in production.
```

Comment:
Probably not for this PR, but I think we're already using this plugin https://github.com/sphinx-contrib/youtube to embed yt videos directly. Having more multi-modal content is important, and I think going through our yt channel and take inventory in terms of what we can cross-link on our docs could be extremely valuable.

Reply:
Good to know. I agree that we could do more cross-linking in general. Thanks!